.changeset/fix-gemini-cli-dependency.md

```
@@ -0,0 +1,5 @@
+---
+"task-master-ai": patch
+---
+
+Fix compatibility with @google/gemini-cli-core v0.1.12+ by updating ai-sdk-provider-gemini-cli to v0.1.1
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)